test/dotnet/install_dotnet_3.sh

```diff
@@ -2,16 +2,10 @@
 
 set -e
 
-# Optional: Import test library
 source dev-container-features-test-lib
 
-check "dotnet sdks" dotnet --list-sdks
-check "some major version of dotnet 3 is installed" bash -c "dotnet --list-sdks |  grep '3\.[0-9]*\.[0-9]*'"
-check "dotnet version 3 installed"  bash -c "ls -l /usr/share/dotnet/sdk | grep '3\.[0-9]*\.[0-9]*'"
+./test.sh
 
-# Verify current symlink exists and works
-check "current link dotnet" /usr/local/dotnet/current/dotnet --info
```

Comment:
Can we validate this path exists else we'd need to update `universal` image? 

We do link to to `/home/...` for the universal image.
https://github.com/devcontainers/images/blob/main/src/universal/.devcontainer/local-features/setup-user/install.sh#L86

Reply:
This test ensures that `/usr/local/dotnet/current/dotnet` exists by calling `test.sh`:
https://github.com/devcontainers/features/blob/b00c0796028f82a4db0b43988cf57e33c5a95691/test/dotnet/install_dotnet_3.sh#L7

`test.sh` verifies that `/usr/local/dotnet/current/dotnet` exists:
https://github.com/devcontainers/features/blob/b00c0796028f82a4db0b43988cf57e33c5a95691/test/dotnet/test.sh#L11-L14

Reply:
Do the checks in `test.sh` cover the case you're concerned about?

Reply:
Sweet, I think I missed that. Thanks for confirming!